src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -3,14 +3,18 @@
     <div class="card-pf-heading f8-card-heading">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="card-pf-heading-details f8-card-heading-details" default-level>
-          <a id="spacehome-codebases-add-space-button" class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-            <i class="pficon pficon-add-circle-o"></i>
-          </a>
-        </div>
-        <div class="card-pf-heading-details f8-card-heading-details" user-level>
+          <div *ngIf="userOwnsSpace">
             <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-                <i class="pficon pficon-add-circle-o"></i>
+              <i id="default-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
             </a>
+          </div>
+          <div class="card-pf-heading-details f8-card-heading-details" user-level>
+            <div *ngIf="userOwnsSpace">
```

Comment:
Using `ngContainer` instead of `div` for all of these ngIf's would be least likely to affect the appearance of the page. Thoughts?

Reply:
Yes, that's true. As it stands there's no difference for many of the components, but perhaps it would be a good way to future proof.